Apps/Examples/Examples/All Examples/CustomAnchoredSymbolExample.swift

```diff
@@ -0,0 +1,321 @@
+import UIKit
+import MapboxMaps
+import Turf
+
+private enum SymbolTailPosition: Int {
+    case left
+    case right
+    case center
+}
+
+private struct DebugFeature {
+    var coordinate: CLLocationCoordinate2D
+    var highlighted: Bool
+    var sortOrder: Int
+    var tailPosition: SymbolTailPosition
+    var label: String
+    var imageName: String
+}
+
+@objc(CustomAnchoredSymbolExample)
+public class CustomAnchoredSymbolExample: UIViewController, ExampleProtocol {
```

Comment:
App code never needs to be public since it won't be consumed by other modules. Keeping it internal unlocks additional compile-time optimizations that are not possible for public constructs.